tests/components/tessie/test_coordinator.py

```diff
@@ -1,107 +1,82 @@
 """Test the Tessie sensor platform."""
 
 from datetime import timedelta
+from unittest.mock import AsyncMock
 
 from freezegun.api import FrozenDateTimeFactory
 from tesla_fleet_api.exceptions import Forbidden, InvalidToken
 
-from homeassistant.components.tessie import PLATFORMS
 from homeassistant.components.tessie.coordinator import (
     TESSIE_FLEET_API_SYNC_INTERVAL,
     TESSIE_SYNC_INTERVAL,
 )
 from homeassistant.config_entries import ConfigEntryState
-from homeassistant.const import STATE_OFF, STATE_ON, STATE_UNAVAILABLE, Platform
+from homeassistant.const import STATE_OFF, STATE_UNAVAILABLE, Platform
 from homeassistant.core import HomeAssistant
 
-from .common import (
-    ERROR_AUTH,
-    ERROR_CONNECTION,
-    ERROR_UNKNOWN,
-    TEST_VEHICLE_STATUS_ASLEEP,
-    setup_platform,
-)
+from .common import ERROR_AUTH, ERROR_CONNECTION, ERROR_UNKNOWN, setup_platform
 
 from tests.common import async_fire_time_changed
 
 WAIT = timedelta(seconds=TESSIE_SYNC_INTERVAL)
 
 
-async def test_coordinator_online(
-    hass: HomeAssistant, mock_get_state, mock_get_status, freezer: FrozenDateTimeFactory
-) -> None:
-    """Tests that the coordinator handles online vehicles."""
-
-    await setup_platform(hass, PLATFORMS)
-
-    freezer.tick(WAIT)
-    async_fire_time_changed(hass)
-    await hass.async_block_till_done()
-    mock_get_status.assert_called_once()
-    mock_get_state.assert_called_once()
-    assert hass.states.get("binary_sensor.test_status").state == STATE_ON
-
-
-async def test_coordinator_asleep(
-    hass: HomeAssistant, mock_get_status, freezer: FrozenDateTimeFactory
-) -> None:
-    """Tests that the coordinator handles asleep vehicles."""
+async def test_coordinator(hass: HomeAssistant, freezer: FrozenDateTimeFactory) -> None:
```

Comment:
Did we loose the online/offline tests now? It's not entirely obvious in here.

Reply:
Yes and No, The status API that was used to identify the online/offline state was removed so there aren't two different code paths to test anymore.

Reply:
It would be good if that would be visible in the test e.g. mock the timestamp and make it go online/offline. Perhaps with a comment for review visibility.

Reply:
Can do, I had originally tried to have that in the PR but was having difficulties with modifying the fixture and mock return value, but can give it another go.

Reply:
Alright got it working.